docs/reference/commandline/daemon.md

```diff
@@ -643,4 +644,20 @@ set like this:
     /usr/local/bin/docker daemon -D -g /var/lib/docker -H unix:// > /var/lib/docker-machine/docker.log 2>&1
 
 
+# Default cgroup parent
```

Comment:
It should work for systemd driver if the argument is a systemd slice.

Sent from my iPhone

> On Dec 19, 2015, at 8:21 PM, moxiegirl <notifications@github.com> wrote:
> 
> In docs/reference/commandline/daemon.md:
> 
> > @@ -643,4 +644,20 @@ set like this:
> >      /usr/local/bin/docker daemon -D -g /var/lib/docker -H unix:// > /var/lib/docker-machine/docker.log 2>&1
> >  
> >  
> > +# Default cgroup parent
> @LK4D4 Is this value only valid when the native.cgroupdriver is cgroupsfs? Or does it work if that value is also systemd?
> 
> —
> Reply to this email directly or view it on GitHub.
>